tiles-generation/docker/vector-tile/entrypoint.sh

```diff
@@ -64,7 +64,7 @@ then
 
     # Upload to S3 Bucket
     currentTime=$(date "+%Y%m%d%H%M%S")
-    /s5cmd cp tiles/ s3://"${TILE_S3_BUCKET}"/"${AREA}"-VT-"${currentTime}"/vector/
+    /s5cmd cp tiles/ s3://"${TILE_S3_BUCKET}"/"${AREA}"-VT-"${currentTime}"/data/
```

Comment:
Will this path change cause any Backward compatibility issue?

Reply:
No, since it's the first time to introduce vector tiles to OpenSearch maps service https://tiles.maps.opensearch.org/data/0/0/0.pbf, also it won't effect raster tiles distribution.